packages/amazonq/.changes/next-release/Feature-84497408-f91e-4a25-93b7-e40cce675b47.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Add getFeature and isEnabled utility methods to FeatureConfigProvider"
```

Comment:
Don't need user-facing changelog items for backend implementations

Reply:
I received a failure from `CI / lint-commits`. May be I can make this a fix instead of feat.

Reply:
Yeah maybe select a different type to use. I wonder if we should a new one? Ex: `impl(auth)`

Reply:
"telemetry" may be an applicable type here